components/org.wso2.bfsi.consent.management.extensions/src/main/java/org/wso2/bfsi/consent/management/extensions/internal/ConsentExtensionsServiceComponent.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.bfsi.consent.management.extensions.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.wso2.bfsi.consent.management.common.config.ConsentManagementConfigurationService;
+import org.wso2.bfsi.consent.management.extensions.common.ConsentExtensionExporter;
+import org.wso2.bfsi.consent.management.service.ConsentCoreService;
+
+/**
+ * The Component class for activating consent extensions osgi service.
+ */
+@Component(
+        name = "org.wso2.bfsi.consent.management.extensions.internal.ConsentExtensionsServiceComponent",
+        immediate = true)
+public class ConsentExtensionsServiceComponent {
+
+    private static final Log log = LogFactory.getLog(ConsentExtensionsServiceComponent.class);
+
+    @Activate
+    protected void activate(ComponentContext context) {
+        log.debug("Consent Extensions component is activated");
+
+        context.getBundleContext().registerService(ConsentExtensionExporter.class.getName(),
+                ConsentExtensionExporter.getInstance(), null);
+        if (log.isDebugEnabled()) {
```

Comment:
This if check seems unnecessary here

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/4/commits/98992be82ffa4483d8c6f51e4dce81ee11bb9d3b